desktop/.project

```diff
@@ -24,7 +24,7 @@
 		<link>
 			<name>assets</name>
 			<type>2</type>
-			<location>/home/finalshare/Documentos/1-Programacao/Arquivos java/KGP/Area de trabalho/kogaplanet-launcher/assets</location>
```

Comment:
Ops, little mistake here, i should to add that little guy to the gitignore file.